### PR TITLE
Patch new sdk url

### LINKED ISSUE #14: cannot find latest release
I have:

**build.yml**
```
- uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
   with:
     version: '272.0.0'
     service_account_key: ${{ secrets.GOOGLE_CLOUD_SERVICE_ACCOUNT_JSON_KEY }}
```

When I look at the logs, I see:

```
(node:7088) UnhandledPromiseRejectionWarning: Error: Failed to find release, os: linux arch: x64 version: 272.0.0
```

Interestingly some of my commands do seem to run okay, though maybe it's the previously cached version?

Cloud run, which was made GA in this version, does _not_ run:

**build.yml**
```
- name: Deploy
   run: gcloud run deploy --image gcr.io/"$GOOGLE_CLOUD_PROJECT"/"$CONTAINER_IMAGE":"$GITHUB_SHA" --platform managed
```

**Result:**
```
ERROR: (gcloud) Invalid choice: 'run'.
This command is available in one or more alternate release tracks.  Try:
  gcloud alpha run
  gcloud beta run
```

### PR DESCRIPTION
**Pull request by dakom**
Fixes #14

### CODE REVIEW

File: setup-gcloud/__tests__/download-util.test.ts [hunk 1]
```diff
@@ -50,7 +50,11 @@ describe('downloadAndExtractTool tests', () => {
...
 
-      const url = await getReleaseURL('linux', 'x86_64', testUtil.TEST_SDK_VERSION);
+      const url = await getReleaseURL(
+        'linux',
+        'x86_64',
+        testUtil.TEST_SDK_VERSION,
+      );
       const extPath = await downloadUtil.downloadAndExtractTool(url);
       expect(extPath).toBeDefined();
       expect(fs.existsSync(extPath)).toBe(true);
```
Comment (24cafeteria) [inline setup-gcloud/__tests__/download-util.test.ts]: expect(fs.existsSync(extPath)).toBe(true)

File: setup-gcloud/__tests__/download-util.test.ts [hunk 2]
```diff
@@ -26,8 +26,8 @@ const tempDir = testUtil.setupTempDir('temp', 'RUNNER_TEMP');
 
 // Import modules being tested after test setup as run.
 import * as downloadUtil from '../src/download-util';
```
Comment (24cafeteria) [inline setup-gcloud/__tests__/download-util.test.ts]: // Import modules being tested after test setup as run.	// Import modules being tested after test setup as run.
import * as downloadUtil from '../src/download-util';	import * as downloadUtil from '../src/download-util';

File: setup-gcloud/__tests__/format-url.test.ts [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Tests client-util.
+ */
+import {TEST_SDK_VERSIONS} from '../src/test-util';
```
Comment (24cafeteria) [inline setup-gcloud/__tests__/format-url.test.ts]: import {TEST_SDK_VERSIONS} from '../src/test-util';

**Comment by craigdbarber** [comment]
Closing this out, see: https://github.com/GoogleCloudPlatform/github-actions/pull/20

**Pull request closed without merging**